libs/shared/src/types/feature-flags/flags.types.spec.ts

```diff
@@ -62,3 +62,15 @@ testFlagEnumValidity(InvalidValueFlagsEnum);
  */
 testFlagEnumValidity(FeatureFlagsKeysEnum);
 testFlagEnumValidity(SystemCriticalFlagsEnum);
+
```

Comment:
Option 1:
Defining a type limiter here that requires us to update this file every time we introduce a new enum (same as with the `testFlagEnumValidity()` function). Reports error before build.

Reply:
Agreed with @ainouzgali to go with this option